### ISSUE TITLE
PSR12.Files.FileHeader locks up when inline comment is the last content in a file

### ISSUE THREAD

**Issue by ADmad** [squizlabs/PHP_CodeSniffer#2619]
`PSR12\Sniffs\Files\FileHeaderSniff` hangs up when checking file with following content

```
<?php
// Do nothing

```

**Comment by dereuromark** [comment]
Is this a dup of https://github.com/squizlabs/PHP_CodeSniffer/issues/2616 ?

**Comment by ADmad** [comment]
No it's not. In this case phpcs hangs and never generates any output.

**Comment by gsherwood** [comment]
Thanks for reporting this. Fix will be in 3.5.1.

**Comment by ttrig** [comment]
If you add another comment below `// Do nothing` (716b619fca624716dfa5b40472b6854754ecf34d), the same issue appear, at least for me.
```php
<?php
// Do nothing
// Do nothing again
```
